File: Program/Java Program/salesmanager/salesManager.java
class SalesManager
{
	double incentive;
	int target;
	
	SalesManager()
	{
		incentive=200;
		target=2;	
	}
	SalesManager(double i,int t)
	{
		System.out.println("Parameterized Constructor Called :");
		incentive=i;
		target=t;	
	}


	
	void setIncentive(double i)
	{
		incentive=i;
	}

	void setTarget(int t)
	{
		target=t;
	}

	double getIncentive()
	{
		return incentive;
	}

	int getTarget()
	{
		return target;
	}

	void display()
	{
		System.out.println("Incentive is :"+getIncentive()+"\nTarget is : "+getTarget());
	}

}
class SaleMain
{
	public static void main(String args[])
	{
		SalesManager s1=new SalesManager();
		s1.display();
		s1.setIncentive(2000);

		s1.setTarget(5);
		s1.display();
		System.out.println(s1);
		SalesManager s2=new SalesManager(3000,10);
		s2.display();



	}
}

File: Program/C++ prog/SalesManager.cpp
#include <stdio.h>
#include <string.h>

class Salemang
{
    int id;
    char name[20];
    double sal;
    double incentive;
    int target;

  public:
    void setId(int a)
    {
        this->id = a;
    }
    void setName(char *nm)
    {
        strcpy(this->name, nm);
    }

    void setSal(double b)
    {
        this->sal = b;
    }
    void setIncentive(double x)
    {
        this->incentive = x;
    }

    void setTarget(int y)
    {
        this->target = y;
    }

    int getId()
    {
        return this->id;
    }
    char *getName()
    {
        return this->name;
    }
    double getSal()
    {
        return this->sal;
    }

    double getIncentive()
    {
        return this->incentive;
    }

    int getTarget()
    {
        return this->target;
    }

    void display()
    {
        printf("Person Id :%d Name is :%s Sal is :%ld Incentive is: %ld Target is:%d", this->id, this->name, this->sal, this->incentive, this->target);
    }
};

int main()
{
    Salemang s1;
    s1.setId(10);
    s1.setName("Kundan");
    s1.setSal(20000);
    s1.setIncentive(1000);
    s1.setTarget(25);
    s1.display();
}